### DISCUSSION TITLE
No root folder found

### DISCUSSION THREAD

**Discussion by markferguson1970** [RandomNinjaAtk/docker-lidarr-extended#167] [General]
Hi - I'm trying to setup through portainer and cannot get lidarr-extended to start. These lines show in the log:

2022-12-19 01:12:43 :: Audio :: 1.0.306 :: ERROR :: No root folder found
2022-12-19 01:12:43 :: Audio :: 1.0.306 :: ERROR :: Configure root folder in Lidarr to continue...

I have /music mapped to a folder which is accessible and writeable. I also can't connect to Lidarr via the gui to configure a root folder. Any ideas? What am I missing? Thanks!

**Comment by RandomNinjaAtk** [comment]
Sounds like you haven't configured all the docker parameters correctly. You should review the readme for the parameters: https://github.com/RandomNinjaAtk/docker-lidarr-extended/blob/main/README.md#parameters

**Reply by markferguson1970** [reply]
Got it. Working now. Thank you!
